eng/versioning/external_dependencies.txt

```diff
@@ -86,45 +86,45 @@ org.slf4j:slf4j-simple;1.7.30
-org.springframework:spring-core;5.2.10.RELEASE
-org.springframework:spring-expression;5.2.10.RELEASE
-org.springframework:spring-jdbc;5.2.10.RELEASE
-org.springframework:spring-jms;5.2.10.RELEASE
-org.springframework:spring-messaging;5.2.10.RELEASE
-org.springframework:spring-tx;5.2.10.RELEASE
-org.springframework:spring-web;5.2.10.RELEASE
-org.springframework:spring-webmvc;5.2.10.RELEASE
+org.springframework.boot:spring-boot-dependencies;2.3.7.RELEASE
+org.springframework.boot:spring-boot-actuator-autoconfigure;2.3.7.RELEASE
+org.springframework.boot:spring-boot-autoconfigure-processor;2.3.7.RELEASE
+org.springframework.boot:spring-boot-autoconfigure;2.3.7.RELEASE
+org.springframework.boot:spring-boot-configuration-processor;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-actuator;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-aop;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-cache;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-data-redis;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-logging;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-test;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-validation;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-web;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter-webflux;2.3.7.RELEASE
+org.springframework.boot:spring-boot-starter;2.3.7.RELEASE
+org.springframework.boot:spring-boot;2.3.7.RELEASE
+org.springframework.data:spring-data-commons;2.3.6.RELEASE
+org.springframework.data:spring-data-mongodb;3.0.6.RELEASE
+org.springframework.data:spring-data-redis;2.3.6.RELEASE
+org.springframework.integration:spring-integration-core;5.3.4.RELEASE
+org.springframework.kafka:spring-kafka;2.5.10.RELEASE
+org.springframework.security:spring-security-config;5.3.6.RELEASE
+org.springframework.security:spring-security-core;5.3.6.RELEASE
+org.springframework.security:spring-security-oauth2-client;5.3.6.RELEASE
+org.springframework.security:spring-security-oauth2-resource-server;5.3.6.RELEASE
+org.springframework.security:spring-security-oauth2-core;5.3.6.RELEASE
+org.springframework.security:spring-security-oauth2-jose;5.3.6.RELEASE
+org.springframework.security:spring-security-web;5.3.6.RELEASE
+org.springframework:spring-beans;5.2.12.RELEASE
+org.springframework:spring-context-support;5.2.12.RELEASE
+org.springframework:spring-context;5.2.12.RELEASE
+org.springframework:spring-core;5.2.12.RELEASE
+org.springframework:spring-expression;5.2.12.RELEASE
+org.springframework:spring-jdbc;5.2.12.RELEASE
+org.springframework:spring-jms;5.2.12.RELEASE
+org.springframework:spring-messaging;5.2.12.RELEASE
+org.springframework:spring-tx;5.2.12.RELEASE
+org.springframework:spring-web;5.2.12.RELEASE
+org.springframework:spring-webmvc;5.2.12.RELEASE
 
 # spring-boot-starter-parent is not managed by spring-boot-dependencies or spring-cloud-dependencies.
 org.springframework.boot:spring-boot-starter-parent;2.3.5.RELEASE
```

Comment:
Please update `spring-boot-starter-parent`'s version. to `2.3.7.RELEASE`, and execute related script.